plugins/grid-bookmark/src/GridBookmarkWidget/components/ImportBookmarks.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useState } from 'react'
+import { observer } from 'mobx-react'
+import { getSession } from '@jbrowse/core/util'
+import AssemblySelector from '@jbrowse/core/ui/AssemblySelector'
+import { FileLocation } from '@jbrowse/core/util/types'
+import { FileSelector } from '@jbrowse/core/ui'
+import { openLocation } from '@jbrowse/core/util/io'
+import {
+  IconButton,
+  Button,
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  Grid,
+  Select,
+  MenuItem,
+  makeStyles,
+  Typography,
+} from '@material-ui/core'
+import CloseIcon from '@material-ui/icons/Close'
+import ImportIcon from '@material-ui/icons/Publish'
+import { GridBookmarkModel } from '../model'
+
+const useStyles = makeStyles(() => ({
+  closeDialog: {
+    position: 'absolute',
+    right: 0,
+    top: 0,
+  },
+  dialogContainer: {
+    margin: 15,
+  },
+  flexItem: {
+    margin: 5,
+  },
+}))
+
+function ImportBookmarks({
+  model,
+  assemblyName,
+}: {
+  model: GridBookmarkModel
+  assemblyName: string
+}) {
+  const classes = useStyles()
+  const [dialogOpen, setDialogOpen] = useState(false)
+  const [location, setLocation] = useState<FileLocation>()
+  const [error, setError] = useState<Error>()
+  const [fileType, setFileType] = useState('BED')
+  const session = getSession(model)
+  const [selectedAsm, setSelectedAsm] = useState<string>(
+    assemblyName || session.assemblyNames[0],
+  )
+
+  return (
+    <>
+      <Button startIcon={<ImportIcon />} onClick={() => setDialogOpen(true)}>
+        Import
+      </Button>
+      <Dialog
+        open={dialogOpen}
+        onClose={() => setDialogOpen(false)}
+        maxWidth="xl"
+      >
+        <DialogTitle>
+          <IconButton
+            className={classes.closeDialog}
+            aria-label="close-dialog"
+            onClick={() => setDialogOpen(false)}
+          >
+            <CloseIcon />
+          </IconButton>
+        </DialogTitle>
+        <DialogContent>
+          <Grid container direction="column" spacing={10}>
+            <Grid item>
+              <Typography>
+                Currently, only simple BED file imports are supported
+              </Typography>
+              <Select
+                className={classes.flexItem}
+                data-testid="selectFileType"
+                value={fileType}
+                disabled
+                onChange={event => setFileType(event.target.value as string)}
+              >
+                <MenuItem value="BED">BED</MenuItem>
+                <MenuItem value="TSV">TSV</MenuItem>
+              </Select>
+            </Grid>
```

Comment:
I'd probably leave this out of the import form until other options are implemented.

Reply:
Removed for now, says to import a BED file just as a text note